keyboards/converter/att56k_usb/att56k.c

```diff
@@ -0,0 +1,18 @@
+#include <stdbool.h>
+
+#include "att56k_if.h"
+#include "att56k.h"
+#include <print.h>
+#include "debug.h"
+#include "util.h"
```

Comment:
Most of these could be moved to the keyboard's header file, if they're even needed here, at all. ```suggestion
#include "att56k.h"
```